### PR TITLE
Make sure latest/oldest events also take into account the id

### LINKED ISSUE #2105: "latest" event isn't actually latest when timestamps are identical
After fixing GH-2016, now it appears that "latest" isn't actually latest. It's not factoring in the `pk`.

Can be seen on https://app.getsentry.com/micronnexus-gmbh/ota2007a-production/group/79814016/events/latest/

### PR DESCRIPTION
**Pull request by mattrobenolt**
Fixes GH-2105

### CODE REVIEW

File: src/sentry/models/event.py [hunk 1]
```diff
@@ -42,6 +43,9 @@ class Event(Model):
 
     objects = BaseManager()
 
+    # Key to use when ordering a list of events manually
+    ordering_key = attrgetter('datetime', 'id')
```
Comment (mattrobenolt) [inline src/sentry/models/event.py]: I just shoved this here, but not sure if you have a preference. @dcramer

Comment (dcramer) [inline src/sentry/models/event.py]: its pretty obscure to have on the model (i also hate adding random things to the model) but I dont have a better suggestion

Comment (mattrobenolt) [inline src/sentry/models/event.py]: Yeah, the only other thing would be to randomly put it in the module? Or in `constants`? But that seems just as obscure too.

Comment (dcramer) [inline src/sentry/models/event.py]: tbh constants is probably ok for this

Comment (mattrobenolt) [inline src/sentry/models/event.py]: Alright, will move to constants.

**Pull request merged**